client/my-sites/stats/controller.jsx

```diff
@@ -186,11 +194,7 @@ export function overview( context, next ) {
 	window.scrollTo( 0, 0 );
 
 	const activeFilter = find( filters(), ( filter ) => {
-		return (
-			context.params.period === filter.period ||
-			context.path.indexOf( filter.path ) >= 0 ||
-			( filter.altPaths && context.path.indexOf( filter.altPaths ) >= 0 )
-		);
+		return context.params.period === filter.period || context.path.indexOf( filter.path ) >= 0;
```

Comment:
We could go further with this refactor and use [`String.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) like so:

```suggestion
		return context.params.period === filter.period || context.path.includes( filter.path );
```